lib/livebook/hubs/team_client.ex

```diff
@@ -41,6 +41,8 @@ defmodule Livebook.Hubs.TeamClient do
   @spec get_secrets(String.t()) :: list(Secret.t())
   def get_secrets(id) do
     GenServer.call(registry_name(id), :get_secrets)
+  catch
+    :exit, _ -> []
```

Comment:
I don't like this because it can now accidentally catch bugs we were not supposed to. Is it necessary? Could we express this as a capability?

Reply:
In development, if you try to open the Team hub page, it crashes. It would only help when you have an outdated hub and you need to delete them.

Reply:
We can remove this now, right? This was needed for offline mode?

Reply:
Yeah, we can.